tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -51,6 +51,10 @@ spec:
       aws eks $ENDPOINT_FLAG --region $(params.region) wait cluster-active --name $(params.cluster-name) 
       aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region)
       # enable PD on the cluster 
+      kubectl api-versions 
+      kubectl api-resources
+      # To do - remove this, temporarily adding because apparently apiserver is not recongnizing deamonset for sometime inconsistently, need to see if livez and readyz are active
```

Comment:
shouldn't we print the resources after sleep then ? so that get we can get better info on whether the resources are ready or not ?

Reply:
After 5mins, it unlikely any of the resources are not ready, I'm mostly interested before sleep because ideally CFN-wait should wait until cluster is ready. Does it makes sense ?

Reply:
got it